apps/ledger-live-mobile/src/dynamicContent/useDynamicContentLogic.ts

```diff
@@ -70,6 +73,7 @@ export const useDynamicContentLogic = () => {
     dispatch(setDynamicContentAssetsCards(assetCards));
     dispatch(setDynamicContentNotificationCards(notificationCards));
     dispatch(setDynamicContentLearnCards(learnCards));
+    dispatch(setIsDynamicContentLoading(false));
```

Comment:
Do we properly handle the cases were the fetch fails, for example if the user doesn't have an internet connection ? We should set the cards as an empty array in these cases

Reply:
https://github.com/user-attachments/assets/e00894ab-fdff-44a3-b883-b2110f9273dd

Reply:
Okay, you don't need the `dispatch(setIsDynamicContentLoading(false));` here anymore as it's in the `finally` now